api-auto-registration/usage.hbs.md

```diff
@@ -127,6 +133,24 @@ spec:
 
 ```
 
+### <a id='troubleshooting'></a>Troubleshooting
+
+Here are some handy commands for debugging or troubleshooting the APIDescriptor CR.
+
+1. Find the status of the APIDescriptor CR
+    ```console
+    kubectl get apidescriptor <api-apidescriptor-name> -oyaml | yq .status.conditions
```

Comment:
We should document with the `--jsonpath` tag whenever we can, we don't know if users have `jq` installed.